Dockerfile.release

```diff
@@ -1,4 +1,4 @@
-# TODO(later): llvm-toolchain-16 has been backported to Bullseye with the 11.9 point release –
+# TODO(later): llvm-toolchain-18 has been backported to Bullseye with the 11.9 point release –
```

Comment:
This is the one place where auto-replace isn't helpful – 19 was only backported to Bookworm, not Bullseye. I think we can just remove the TODO here, the slightly easier setup process isn't that important.